cookbooks/aws-parallelcluster-environment/test/controls/cloudwatch_spec.rb

```diff
@@ -14,9 +14,9 @@
   end
 
   describe 'Check the presence of the cloudwatch package gpg key'
-  # In Ubuntu 20.04 due to environment variable the keyring is placed under home of the user ubuntu with the permission of root
+  # In Ubuntu >20.04 due to environment variable the keyring is placed under home of the user ubuntu with the permission of root
   ubuntu2004 = os_properties.ubuntu2004?
-  keyring = os_properties.ubuntu2004? && !os_properties.on_docker? ? '--keyring /home/ubuntu/.gnupg/pubring.kbx' : ''
+  keyring = (os_properties.ubuntu2004? || os_properties.ubuntu2204?) && !os_properties.on_docker? ? '--keyring /home/ubuntu/.gnupg/pubring.kbx' : ''
```

Comment:
if you know this is going to carry forward I would do a forward looking check like this if possible

```
node['platform_version'].to_i >= 20
```